financial-templates-lib/Logger.js

```diff
@@ -158,6 +160,12 @@ if (process.env.SLACK_WEBHOOK) {
   );
 }
 
+// If all the required environment variables for twilio are added, add the transport
+if (process.env.TWILIO_SID && process.env.TWILIO_AUTH && process.env.DRI_NUMBER1 && process.env.TWILIO_FROM_NUMBER) {
```

Comment:
nit: don't we do this same check in the `RoboCaller` class? Just curious if there's a simple way to avoid having to check this in two places?

Reply:
you could create a separate module, "Transports.js" perhaps, that has a simple getter() which just checks which transports are available depending on which .env variables are filled out correctly.

Reply:
would have the benefit of isolating all "dotenv" calls to one module

Reply:
I tried to refactor this to have one check in the `RoboCaller` that has a method that can be called but it is inaccessible from within the `Logger.js` because the object has not been initialized as of yet.

Reply:
@nicholaspai this is a good idea. I want to separate that to a different PR though because it will change the slack implementation as well. I created an issue for it: #1106

Reply:
Does this mean that you *won't* be checking in the `RoboCaller` class, itself? Would the environment variables be passed from this new module all the way down to the `RoboCaller` class, so it doesn't need to look them up?

Reply:
ye, I think this is what @nicholaspai is suggesting, right?

Reply:
SGTM